.travis.yml

```diff
@@ -107,3 +107,9 @@ jobs:
               on:
                   tags: true
                   all_branches: true
+
+      after_deploy:
```

Comment:
Not that build stage, it should be part of "DOCKER_REPOSITORY=mendersoftware/mender-conductor" build